parity/rpc.rs

```diff
@@ -253,7 +255,16 @@ pub fn new_ipc<D: rpc_apis::Dependencies>(
 		}
 	}
 
-	match rpc::start_ipc(&conf.socket_addr, handler, rpc::RpcExtractor) {
+	// some validations ..
+	let chmod = conf.chmod;
+	let chmod = u16::from_str_radix(&chmod, 8)
+		.map_err(|e| format!("Invalid octal value: {}", e))?;
+
+	if chmod == 0 || chmod > 4095 {
```

Comment:
```suggestion
	if chmod == 0 || chmod > 0o7777 {
```

Reply:
yeah forgot I could just do this, nice catch!